### PR TITLE
Tmp fix for current period usage

### PR DESCRIPTION
**Pull request by sanderpick**
Fix for a nit I noticed while testing on staging. Usage for monthly incremental products was not showing usage for the current day, only historical for the month. Was only noticeable across multiple days of usage, which my staging account has.

### CODE REVIEW

File: api/billingd/service/service.go [hunk 1]
```diff
@@ -475,7 +476,7 @@ func (s *Service) segmentNewCustomer(cus *Customer) {
 				Set("parent_key", cus.ParentKey).
 				Set("customer_id", cus.CustomerID).
 				Set("account_type", cus.AccountType).
-				Set("hub_signup", "true"),
+				Set(s.config.SegmentPrefix+"signup", "true"),
```
Comment (sanderpick) [inline api/billingd/service/service.go:479]: @andrewxhill I snuck this in here.

File: api/billingd/service/service.go [hunk 2]
```diff
@@ -827,20 +828,20 @@ func (s *Service) GetCustomerUsage(
...
-				total += u.Total
+				free, err := s.getPeriodUsageItem(u.FreeItemID)
+				if err != nil {
+					return nil, err
+				}
+				paid, err := s.getPeriodUsageItem(u.PaidItemID)
+				if err != nil {
+					return nil, err
+				}
+				total += (free.TotalUsage + paid.TotalUsage) * product.UnitSize
```
Comment (sanderpick) [inline api/billingd/service/service.go:844]: This logical will have to change when I add the usage by date range functionality, but this works for the current billing period (month).

File: core/usage_interceptor.go [hunk 3]
```diff
@@ -100,7 +100,7 @@ func (t *Textile) preUsageFunc(ctx context.Context, method string) (context.Cont
 	if err != nil {
 		if strings.Contains(err.Error(), mongo.ErrNoDocuments.Error()) {
 			opts := []billing.Option{
-				billing.WithEmail(account.Owner().Email),
+				billing.WithEmail(account.User.Email),
```
Comment (sanderpick) [inline core/usage_interceptor.go:103]: @andrewxhill This will use the email for the calling org member.

Comment (andrewxhill) [reply core/usage_interceptor.go:103]: Ah nice

**Review by andrewxhill — APPROVED** [review]

**Pull request merged**